client/src/js/collections/DogsCollection.js

```diff
@@ -1,14 +1,67 @@
+var $ = require('jquery');
 var Backbone = require('backbone');
 
+var statusMap = {
+  1: {
+    color: 'green',
+    name: 'in office'
+  },
+
+  2: {
+    color: 'yellow',
+    name: 'on a walk'
+  },
+
+  3: {
+    color: 'red',
+    name: 'missing'
+  }
+};
+
 var Dog = Backbone.Model.extend({
   defaults: {
     display_id: 0,
     name: '',
     short_description: '',
     checked_in: false,
-    avatar: ''
+    avatar: '',
+    current_status: 1,
+    current_status_verbose: 'in office',
+    current_status_color: 'green'
```

Comment:
Nice job with this! I like how you've separated out the status from the verbose details.

Just a few things:

* Remember that everything in programming is [zero based](https://en.wikipedia.org/wiki/Zero-based_numbering), so we'll probably want to start with 0 for `current_status`.
* We should probably have the default for dogs `current_status` be null. This way, the dog will not appear to be `in office/on a walk/missing` if the dog is checked out.
* We should only save the `current_status` on the dog and move the verbose details to a function. Using the dog's `current_status`, we can lookup the other details just like you already have, except without needing to save that data on the dog.